src/main/webapp/map.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<head>
+	<title>Popular places to eat in Singapore</title>
+  <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBawpEnaMnt7vwsqGV_SaUEW7oppEDHJvg&libraries=visualization">
+  </script>
+	<script type="text/javascript" src="/js/popularitymap-loader.js"></script>
+	<style>
+		#map {
```

Comment:
see the comments on the previous PR - these seems to be incorrectly formatted, make sure that your editor is using spaces for indentation, not tabs.